code/modules/medical/genetics/bioEffects/harmful.dm

```diff
@@ -707,31 +707,40 @@
 	stability_loss = -10
 	var/const/radius = 2
 	icon_state  = "bad"
+	var/roundedmultremainder
 
-	OnLife()
+	OnLife(var/mult)
 		..()
-		if (prob(50))
-			return
```

Comment:
Lag-compensation for this one could/should have just been adjusting this prob roll I feel...

Reply:
it's a lot more destructive with the ability to make multiple in one cycle from my testing.

Reply:
Please change it to just adjust the prob roll